src/sentry/migrations/0208_backfill_project_transaction_thresholds.py

```diff
@@ -0,0 +1,74 @@
+# Generated by Django 1.11.29 on 2021-06-15 13:20
+import logging
+from enum import Enum
+
+from django.db import migrations
+
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+class TransactionMetric(Enum):
+    DURATION = 1
+    LCP = 2
+    FCP = 3
+
+
+def migrate_project_transaction_thresholds(apps, schema_editor):
+    """
+    Migrate an org's apdex thresholds (if they exist) to the
+    projects transaction thresholds in the org.
+    """
+    OrganizationOption = apps.get_model("sentry", "OrganizationOption")
+    Project = apps.get_model("sentry", "Project")
+    ProjectTransactionThreshold = apps.get_model("sentry", "ProjectTransactionThreshold")
+
+    for option in RangeQuerySetWrapperWithProgressBar(OrganizationOption.objects.all()):
+        if option.key != "sentry:apdex_threshold":
+            continue
+
+        for project in Project.objects.filter(organization_id=option.organization_id):
+            try:
+                # It is possible that an organization has already set a ProjectTransactionThreshold
+                # and we wouldn't want to overwrite it.
+                ProjectTransactionThreshold.objects.get_or_create(
```

Comment:
Do you need to worry about organizations that set a threshold after the migration runs? Will those new writes update both the organization and the projects?

Reply:
Hmm good callout, yeah we'd want to capture those updates. I'll make a separate PR for that and plan to merge that before this migration so there are no data gaps.

Reply:
If you start writing to both places first, then backfill you won't miss any writes.

Reply:
Added a PR https://github.com/getsentry/sentry/pull/26642 here for it